kolibri/core/assets/src/utils/appCapabilities.js

```diff
@@ -0,0 +1,16 @@
+import urls from 'kolibri.urls';
+import plugin_data from 'plugin_data';
+
+const appCapabilities = plugin_data.appCapabilities || {};
+
+export const can = {
+  launchIntent: appCapabilities.launch_intent,
```

Comment:
is this capability really for generic launching of intents?

I would have expected one of:

* `share_resource`
* `launch_android_intent`

or something more specific

Reply:
Yeah, there could be other sorts of intents at some point, so my vote would be for something like "share_file".

Reply:
I had like 7 different names in there for different functions trying different approaches to the sharing until I finally got one to work reliably. Will be migrating it into the app plugin anyway, and will give it consistent naming there.